### ISSUE TITLE
Value side of an invalid property

### ISSUE THREAD

**Issue by sandrojohanides** [braver/SublimeSass#41]
`#{$prop}` is a dynamic property, but the value side is incorrectly highlighted. Also happens with invalid properies such as `a:`, but **NOT** `x:`.

![screen shot 2018-11-26 at 12 56 35](https://user-images.githubusercontent.com/8327913/49012801-d5cbaf80-f17a-11e8-99ef-8917d661e617.png)
```scss
&[data-color="#{$i - 1}"] {
	#{$prop}: var(--#{$color}-primary, color(primary, unthemed));
}
```

**Comment by braver** [comment]
Ha, that's a nice challenge, I didn't think of dynamic properties yet. Thanks for bringing it up!

**Comment by sandrojohanides** [comment]
Also the question for me at least is if it should support invalid properties with correct syntax, but I don’t see why not.

**Comment by braver** [comment]
well, for the highlight to work without actually compiling everything first, I lean on lists of known property names. also, you want the highlighting to catch typos in property names (e.g. `colour`). so, invalid properties will probably break the highlighting, but I really want to support interpolation here.

**Comment by braver** [comment]
This is fixed in 810c6438c204178255c4039ec5fab3e97b3cfba6, where interpolation now works for selectors and property-names. There are probably some esoteric expressions that trip it up, if you find any let me know!
